### PR TITLE
[6.3.x] Bug 1316566 - User can not create projects in any other repository than master

### PR DESCRIPTION
**Pull request by Rikkola**
Ticket for this branch is here: https://bugzilla.redhat.com/show_bug.cgi?id=1316566

Again had to do some adjustments.

(cherry picked from commit 5c97312841abdbe3cedc83f5830c14c61a2c5308)

https://github.com/droolsjbpm/guvnor/pull/286
https://github.com/droolsjbpm/kie-wb-common/pull/310
https://github.com/droolsjbpm/drools-wb/pull/144
https://github.com/droolsjbpm/jbpm-console-ng/pull/365
https://github.com/droolsjbpm/kie-wb-distributions/pull/229

### CODE REVIEW

File: kie-wb-common-widgets/kie-wb-common-ui/src/test/java/org/kie/workbench/common/widgets/client/datamodel/PackageDataModelMethodsTest.java [hunk 1]
```diff
@@ -43,6 +44,7 @@
 import static org.kie.workbench.common.widgets.client.datamodel.PackageDataModelOracleTestUtils.*;
 import static org.mockito.Mockito.*;
 
+@Ignore("TODO: REVMOE BEFORE COMMIT")
```
Comment (manstis) [inline kie-wb-common-widgets/kie-wb-common-ui/src/test/java/org/kie/workbench/common/widgets/client/datamodel/PackageDataModelMethodsTest.java]: I guess you never removed this?

**Comment by manstis** [comment]
+1 to merge (subject to ```PackageDataModelMethodsTest``` tests being reinstated)

File: kie-wb-common-widgets/kie-wb-common-ui/src/test/java/org/kie/workbench/common/widgets/client/datamodel/PackageDataModelMethodsTest.java (hunk 1 above)
Comment (Rikkola) [inline kie-wb-common-widgets/kie-wb-common-ui/src/test/java/org/kie/workbench/common/widgets/client/datamodel/PackageDataModelMethodsTest.java]: Yep, thanks. I have a different Java version and this keeps failing.

**Comment by Rikkola** [comment]
Removed the @Ignore from PackageDataModelMethodsTest.

**Comment by Rikkola** [comment]
jenkins please retest

**Comment by manstis** [comment]
@Rikkola Is this OK for 6.3.x branch? The BZ lacks sufficient acks.

**Comment by Rikkola** [comment]
@manstis There is one ticket per branch. Looks like, because of cherry-picking, I have the wrong ticket number here. 1316224 is probably for master or 6.4.x.

https://bugzilla.redhat.com/show_bug.cgi?id=1316566

**Comment by Rikkola** [comment]
Nope, sorry the topic is wrong. Commits have the right number. Copy error when I made the PR titles.

**Comment by manstis** [comment]
+1 to merge.. @csadilek all yours!

**Pull request merged**